server/athenian/api/controllers/histograms_controller.py

```diff
@@ -41,12 +41,15 @@ async def calculate_for_set_histograms(service, repos, withgroups, labels, jira,
                 bins=h.bins,
                 ticks=tuple(h.ticks) if h.ticks is not None else None,
             )].append(h.metric)
+        calculator = await get_calculator_for_user(
```

Comment:
Let's consider `gather()`-ing this with the previous coroutines, e.g. fetching the release settings is independent of finding the calculator and can run in parallel.